src/layer.js

```diff
@@ -145,50 +145,63 @@ export default class Layer {
    * @return {Promise} [will resolve once image src has been retrieved and dimensions calculated]
    */
   calculateSize() {
-    // Create image element
-    this.imageEl = new Image();
-    this.imageEl.crossOrigin = 'Anonymous';
-    this.imageEl.src = this.data.image;
+    var _this = this;
```

Comment:
you don't need to remap the context when using es6 shorthand functions as they are lexically scoped, you can remove this and just use `this` directly inside the function block